### ISSUE TITLE
Linebreaks after 'get' or 'set' in class bodies are not parsed correctly (T7533)

### ISSUE THREAD

**Issue by motiz88** [babel/babylon#157]
> Issue originally made by @bakkot (as T7533 and later babel/babel#4432)
### Input code

```
class A {
  get
    x(){}
}
```
### Description

This code gives a parse error without the class-properties-transform plugin, and is parsed as a class containing a single method `x` with it. Neither is correct; it's valid syntax for a class with a getter (not method) for 'x'.

**Comment by motiz88** [comment]
All parsers on ast-explorer.net, [except for TypeScript](https://astexplorer.net/#/TUQ7X6JE0X), disagree with [Babylon's](https://astexplorer.net/#/FfgQs2zhyt/1) parse here.

**Comment by bakkot** [comment]
The issue is larger than I initially reported. All of the following are incorrectly rejected:

```js
class C {
  x
  (){}
}
```

```js
class C {
  ''
  (){}
}
```

```js
class C {
  ['']
  (){}
}
```

```js
class C {
  get
  (){}
}
```

```js
class C {
  set
  (){}
}
```

```js
class C {
  static
  (){}
}
```

```js
class C {
  async
  (){}
}
```

etc., which define methods named 'x', '', 'get', 'set', 'static', or 'async'.

**Comment by danez** [comment]
Fixed as of 6.17.0
